### PR TITLE
Expose load provider end point in client and change the load provider…

### PR DESCRIPTION
**Pull request by paimaam**
Changed the response type of load config and expose load config end point

### CODE REVIEW

**Comment by NakulSeerpu** [comment]
Could you specify what is the change in response type

File: sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs [hunk 1]
```diff
@@ -88,6 +89,9 @@ private static async Task<string> Run(AppConfiguration configuration)
 
                 var citizenDocumentConfiguration = await citizenDocumentClient.CreateProviderConfiguration(requestToupload).ConfigureAwait(false);
                 configId = citizenDocumentConfiguration.ConfigurationId.Value;
+
+                var loadedConfigurations = await citizenDocumentClient.LoadProviderConfiguration().ConfigureAwait(false);
+                Console.WriteLine(loadedConfigurations.First().ConfigId);
```
Comment (NakulSeerpu) [inline sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs]: How about, firstdefault.

Comment (paimaam) [reply sample/Kmd.Logic.CitizenDocuments.Client.Sample/Program.cs]: changed

**Review by Araballi — APPROVED** [review]
LGTM

**Pull request merged**